decidim-core/lib/decidim/assets/tailwind/tailwind.config.js.erb

```diff
@@ -47,21 +47,32 @@ module.exports = {
         3: "#E1E5EF",
         4: "#242424",
         5: "#F6F8FA",
-        6: "#D3D5D9"
+        6: "#D3D5D9",
```

Comment:
This should be removed and its usages replaced with `neutral-300` (#d4d4d4 and #d3d5d9 are almost the same color as far as I see) 

<img width="376" height="137" alt="Image" src="https://github.com/user-attachments/assets/0afb8f75-92f1-45d6-96e5-7342922caf0e" />

<img width="356" height="148" alt="Image" src="https://github.com/user-attachments/assets/40e8e212-1801-4cd5-b823-601680fec7b8" />

https://www.color-hex.com/color/d3d5d9
https://www.color-hex.com/color/d4d4d4